app/assets/javascripts/discourse/app/instance-initializers/auth-complete.js

```diff
@@ -53,7 +53,11 @@ export default {
                 ...properties,
               };
 
-              router.trigger("showLogin", props);
+              router.transitionTo("login").then((login) => {
+                for (const [key, value] of Object.entries(props)) {
+                  login.controller.set(key, value);
+                }
+              });
```

Comment:
I think we can be a little less verbose with 


```suggestion
              router.transitionTo("login").then(({ controller }) => {
                controller.setProperties(...props);
              });
```